rpcs3/Emu/Cell/PPULLVMRecompilerCore.cpp

```diff
@@ -5207,11 +5207,11 @@ void Compiler::CreateBranch(llvm::Value * cmp_i1, llvm::Value * target_i32, bool
 			}
 
 			SetPc(target_i32);
-//			Function *fn = m_module->getFunction(fmt::format("function_0x%08X", target_address));
+			Function *fn = m_module->getFunction(fmt::format("function_0x%08X", target_address));
```

Comment:
Initially I wanted to make LLVM compile several function at once, if a block is directly calling a function then it is detected by AnalyseBlock and both the block and the called function are compiled to avoid the cost of indirect function call when possible.
(Note that sometimes the function pointer is not know at compile time and for such case indirect function call is still needed).
However the overhead of indirect function call drastically lowered when I removed the use of some mutexes in the recompilation engine and used virtual memory instead. Direct function call is still cheaper than indirect function call but it's not as necessary as it was back in July.

At the moment the code builds a single function and thus "m_module->getFunction(fmt::format("function_0x%08X", target_address));" will always returns nullptr so the true branch of the if (fn) is never executed. It doesn't matter if the code is commented or not, although it might help to add some comment above if the code is uncommented.